LayerClient/LayerClient.py

```diff
@@ -319,6 +320,25 @@ def send_announcement(self, sender, recipients, message_parts,
             )
         )
 
+    def mark_delivery_receipt_message(self, message_uuid='', type='sent'):
+        """
+        Mark delivery receipt for message
+
+        Parameters:
+        - `message_uuid`: The uuid of the message read
+		- `type`: receipt message type
```

Comment:
Looks like you have some tabs here; would you mind converting your tabs to 4 spaces?

Convenient way to do this in vim would be 

    :set tabstop=4 shiftwidth=4 expandtab
    :retab